docs/ansible/roles/netbox/getting-started.rst

```diff
@@ -15,8 +15,8 @@ General deployment notes
 ------------------------
 
 The application will be by default deployed both on ``ipam.{{ ansible_domain
-}}`` as well ass ``dcim.{{ ansible_domain }}`` DNS domains, for convenience.
```

Comment:
Whoops. :-) I think that my keyboard is the culprit - I noticed occasional repeats of keys which are unintended. I'm currently using [SteelSeries Merc Stealth](https://steelseries.com/blog/steelseries-discontinued-gaming-keyboards-130) but it seems that the keyboard is starting to show its age... Any suggestions for a replacement?

Reply:
There's that company named "Microsoft", their cheapest keyboards are pretty good. Something to do with wanting their logo on as many desks as possible... I don't know why, I think they also sell mice.

Mine doubles as an ashtray as well.

Reply:
> Whoops. :-)

Happens :)